### PR TITLE
API Remove deprecated code

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10501

### CODE REVIEW

File: code/LeftAndMain.php [hunk 1]
```diff
@@ -1681,7 +1663,6 @@ public function currentPageID()
             return $this->getRequest()->param('ID');
         }
 
-        /** @deprecated */
```
Comment (emteknetnz) [inline code/LeftAndMain.php:1684]: Could you leave this in - we'll need to separately what the `".currentPage"` functionality was - originally it looks like someone intended for it to be removed

File: code/LeftAndMain.php [hunk 2]
```diff
@@ -1698,7 +1679,6 @@ public function setCurrentPageID($id)
     {
         $this->pageID = $id;
         $id = (int)$id;
-        /** @deprecated */
```
Comment (emteknetnz) [inline code/LeftAndMain.php]: Leave this in

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: code/LeftAndMain.php (hunk 1 above)
Comment (sabina-talipova) [reply code/LeftAndMain.php:1684]: Done

File: code/LeftAndMain.php (hunk 2 above)
Comment (sabina-talipova) [reply code/LeftAndMain.php]: Done

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**